source/cgame/cg_events.cpp

```diff
@@ -589,6 +589,12 @@ static void DoEntFX( const SyncEntityState * ent, u64 parm, Vec4 color, StringHa
 	PlaySFX( sfx, PlaySFXConfigPosition( ent->origin ) );
 }
 
+static PlayingSFXHandle PlayViewerSound( StringHash sfx, SyncEntityState * ent, bool viewer ) {
+	return 	viewer ?
```

Comment:
PlayEntityOrFirstPersonSFX

const SyncEntityState

no need to pass viewer in as an arg because it's just ISVIEWERENTITY( ent->number )

good change though